Tasks/AppCenterDistributeV3/Tests/L0UtilsUnitTests.ts

```diff
@@ -0,0 +1,27 @@
+import * as path from 'path';
+import * as assert from 'assert';
+import {spawnSync} from 'child_process';
+
+export function utilsUnitTests() {
+    describe('Utils unit tests', function () {
+        describe('findCommonParent', function() {
+            it('returns containing directory for single file', function() {
+                const tp = path.join(__dirname, 'UnitTests', 'Utils', 'FindCommonParentForSingleFileReturnsContainingDirectory.js');
+                const spawn = spawnSync('node', [tp]);
```

Comment:
It appears we're calling `spawnSync` without any `timeout`. By default, the `timeout` for these methods are `undefined`: https://nodejs.org/api/child_process.html#child_process_child_process_spawnsync_command_args_options

It appears this could cause a hang in the test execution unless we supply a timeout period, from my understanding of the documentation:
> The child_process.spawnSync() method is generally identical to child_process.spawn() with the exception that the function will not return until the child process has fully closed. When a timeout has been encountered and killSignal is sent, the method won't return until the process has completely exited.

Should we supply a timeout to `spawnSync()`?